frontend/package.json

```diff
@@ -67,15 +67,7 @@
-    "watch-css": "yarn compile-scss && node-sass --include-path ./src/scss --include-path ./node_modules/uswds/dist/scss -o src/styles/ --watch --recursive src/scss/App.scss",
-    "deploy-azure-dev": "REACT_APP_OKTA_URL=https://dev.simplereport.gov REACT_APP_APPINSIGHTS_KEY=29a75c50-8485-4d80-8e1b-41b1a091abb6 REACT_APP_BACKEND_URL=https://dev.simplereport.gov/api REACT_APP_BASE_URL=https://dev.simplereport.gov PUBLIC_URL=/app/ yarn build && az storage blob upload-batch -s build/ -d '$web' --account-name simplereportdevapp --destination-path '/app'",
-    "deploy-azure-test": "REACT_APP_OKTA_URL=https://test.simplereport.gov REACT_APP_BACKEND_URL=https://test.simplereport.gov/api REACT_APP_BASE_URL=https://test.simplereport.gov PUBLIC_URL=/app/ yarn build && az storage blob upload-batch -s build/ -d '$web' --account-name simplereporttestapp --destination-path '/app'",
-    "deploy-azure-pentest": "REACT_APP_OKTA_URL=https://pentest.simplereport.gov REACT_APP_BACKEND_URL=https://pentest.simplereport.gov/api PUBLIC_URL=/app/ yarn build && az storage blob upload-batch -s build/ -d '$web' --account-name simplereportpentestapp --destination-path '/app'",
-    "deploy-azure-training": "REACT_APP_OKTA_URL=https://training.simplereport.gov REACT_APP_BACKEND_URL=https://training.simplereport.gov/api REACT_APP_IS_TRAINING_SITE=true PUBLIC_URL=/app/ yarn build && az storage blob upload-batch -s build/ -d '$web' --account-name simplereporttrainingapp --destination-path '/app'",
-    "deploy-azure-demo": "REACT_APP_OKTA_URL=https://demo.simplereport.gov REACT_APP_BACKEND_URL=https://demo.simplereport.gov/api REACT_APP_BASE_URL=https://demo.simplereport.gov PUBLIC_URL=/app/ yarn build && az storage blob upload-batch -s build/ -d '$web' --account-name simplereportdemoapp --destination-path '/app'",
-    "deploy-azure-stg": "REACT_APP_OKTA_URL=https://stg.simplereport.gov REACT_APP_BACKEND_URL=https://stg.simplereport.gov/api PUBLIC_URL=/app/ yarn build && az storage blob upload-batch -s build/ -d '$web' --account-name simplereportstgapp --destination-path '/app'",
-    "deploy-azure-prod": "REACT_APP_APPINSIGHTS_KEY=ec65665e-358e-4f07-9292-3d80f23bcc3b REACT_APP_OKTA_URL=https://simplereport.gov REACT_APP_BASE_URL=https://simplereport.gov REACT_APP_BACKEND_URL=https://simplereport.gov/api PUBLIC_URL=/app/ yarn build && az storage blob upload-batch -s build/ -d '$web' --account-name simplereportprodapp --destination-path '/app'",
-    "maintenance-azure-beta": "mkdir -p /tmp/maintenance && /bin/cp -f public/maintenance.html /tmp/maintenance/index.html && az storage blob upload-batch -s /tmp/maintenance/ -d '$web' --account-name usdssimplereportbeta --destination-path '/'"
+    "watch-css": "yarn compile-scss && node-sass --include-path ./src/scss --include-path ./node_modules/uswds/dist/scss -o src/styles/ --watch --recursive src/scss/App.scss"
```

Comment:
And this is why not allowing trailing commas is annoying. (I don't even know if this file allows them, at this point, it's totally reasonable to take it out by reflex if it does, it's just... a pet peeve.)

Reply:
json doesn't allow it but I agree and prettier defaults to trailing commas in JS now!